lib/booking/ui/pages/admin_pages/add_edit_room_page.dart

```diff
@@ -67,10 +69,20 @@ class AddEditRoomPage extends HookConsumerWidget {
                     height: 50,
                   ),
                   managerList.when(
-                    data: (List<Manager> data) => ScrollManagerChips(
-                      data: data,
-                      managerId: managerId,
-                      managerIdNotifier: managerIdNotifier,
+                    data: (List<Manager> data) => AdminScrollChips(
+                      isEdit: isEdit,
+                      dataKey: dataKey,
+                      pageStorageKeyName: "manager_list",
+                      builder: () => data.map(
```

Comment:
Can't you make a builder for e instead of data ?

Reply:
Yes it's a good idea.